recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.4.rc3" %}
+{% set source_tag = "2.4-rc3" %}
+
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/{{ source_tag }}.tar.gz
+  sha256: 48a52af8f9da6deef5c01d26a30ba4bdc19929999d6680dab6461b92b5a63bd7
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - {{ compiler('cxx') }}  # [not osx]
+    - clang_osx-64  # [osx]
+    - clangxx_osx-64  # [osx]
+
+  host:
+    - python
+    - numpy
```

Comment:
Do you have a minimum supported `numpy` version?

Reply:
1.6 but that's really old, and setting `numpy 1.6.*` triggers python 2.7

Reply:
So we don't really have a minmum version restriction

Reply:
Oh ok. So the `conda-forge` defaults should be fine. Just wanted to make sure a newer version wasn't needed.

ref: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/48c17525eac458ee59a7f9d4ba16bf5e7407de16/recipe/conda_build_config.yaml#L353-L355

Reply:
Is there a need to make it explicit?

Reply:
Nope. We set the minimums to `1.9` on Unix and `1.11` on Windows if nothing else is specified.